build.sbt

```diff
@@ -266,138 +266,31 @@ def macroCompileSettings: Seq[Setting[_]] = Seq(
   }
 )
 
-val version150 = "1.5.0"
-val version151 = "1.5.1"
-val version160 = "" // TODO: Set to 1.6.0 when it's out, also remove the Scala 2.13 exclusion
+val previousVersions = Seq("1.6.0")
 
-def noMima = mimaPreviousArtifacts := Set.empty
-def mimaSettings(since: String = version150): Seq[Setting[_]] = {
+val noMima = mimaPreviousArtifacts := Set.empty
+val mimaSettings: Seq[Setting[_]] = {
   import ProblemFilters.exclude
   import sbt.librarymanagement.SemanticSelector
   import sbt.librarymanagement.VersionNumber
-  val versions = if (since == "") Nil else Seq(since)
   Seq(
-    mimaPreviousArtifacts := {
+    mimaPreviousArtifacts := previousVersions.map { version =>
+      val suffix   = if (crossPaths.value && !sbtPlugin.value) s"_${scalaBinaryVersion.value}" else ""
+      val moduleID = organization.value % s"${moduleName.value}$suffix" % version
+
+      // For sbt plugins if that is the case for the current subproject
       val sbtBV   = (sbtBinaryVersion in pluginCrossBuild).value
       val scalaBV = (scalaBinaryVersion in pluginCrossBuild).value
-      scalaVersionFilter(scalaBinaryVersion.value, versions).map { version =>
-        val suffix   = if (crossPaths.value && !sbtPlugin.value) s"_${scalaBinaryVersion.value}" else ""
-        val moduleID = organization.value % s"${moduleName.value}$suffix" % version
-        if (sbtPlugin.value) Defaults.sbtPluginExtra(moduleID, sbtBV, scalaBV)
-        else moduleID
-      }.toSet
-    },
-    mimaPreviousArtifacts := {
-      if (VersionNumber(scalaVersion.value).matchesSemVer(SemanticSelector(">=2.13"))) Set.empty
-      else mimaPreviousArtifacts.value
-    },
+
+      if (sbtPlugin.value) Defaults.sbtPluginExtra(moduleID, sbtBV, scalaBV)
+      else moduleID
+    }.toSet,
     mimaBinaryIssueFilters ++= Seq(
-      ProblemFilters.exclude[Problem]("com.lightbend.lagom.internal.*"),
```

Comment:
Should've kept this one.  Components too, according to https://github.com/lagom/lagom/pull/1483/commits/53371cf40c62ee0128da5abc319388a76a851d8d